modules/admin/proc-rhdh-deploy-eks-using-helm.adoc

```diff
@@ -1,34 +1,45 @@
-[id='proc-rhdh-deploy-eks-using-helm_{context}']
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-rhdh-integration-aws.adoc
+//
 
+[id="proc-rhdh-deploy-eks-using-helm_{context}"]
 = Deploying {product} in {eks-name} ({eks-short}) using Helm Chart
 
 When you deploy {product-short} in {eks-name} ({eks-short}) using Helm Chart, it orchestrates a robust development environment within the AWS ecosystem.
 
 .Prerequisites
 
 * You have an {eks-short} cluster with AWS Application Load Balancer (ALB) add-on installed. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html[Application load balancing on Amazon {product-short}] and https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html[Installing the AWS Load Balancer Controller add-on].
+* You have installed the link:https://kubernetes.io/docs/reference/kubectl/[`kubectl` CLI].
+* You started a `kubectl` session, with `developer` or `admin` permissions.
```

Comment:
do you think this would be more clear as...
```suggestion
* You are logged into your cluster using `kubectl`, and have `developer` or `admin` permissions.
```